doc/user/content/transform-data/patterns/top-k.md

```diff
@@ -100,3 +100,27 @@ FROM cities
 ORDER BY state, pop DESC;
 ```
 Note that the `ORDER BY` clause should start with the expressions that are in the `DISTINCT ON`.
+
+## Group size hints
+
+When using either the above `LATERAL` subquery pattern or `DISTINCT ON`, we recommend specifying a [group size hint](/transform-data/optimization/#query-hints) to improve memory usage. For example:
```

Comment:
Since here we are referring to query hints in general here, I'd refer to their definition/syntax, e.g.:
```suggestion
When using either the above `LATERAL` subquery pattern or `DISTINCT ON`, we recommend specifying [query hints](/select/#query-hints) to improve memory usage. For example:
```

Reply:
Done